### DISCUSSION TITLE
Possible to use with Matter Server addon?

### DISCUSSION THREAD

**Discussion by timkolloch** [t0bst4r/home-assistant-matter-hub#353] [Q&A] [answered]
Hey guys, I am using my HomeAssistant as my Matter server as well. 
As far as I understand the addon should work anyway, am I correct?

So I should be able to connect my Echo Dot (newest generation) to my HomeAssistant like that and then control my lights without any hassle or is there something I need to keep in mind? 

Thank you all in advance!

**Comment by Apollon77** [comment, marked as answer]
Should work.

**Comment by timkolloch** [comment]
> Should work.

Thank you!
